scripts/CMSRunAnalysis.py

```diff
@@ -616,7 +632,9 @@ def executeScriptExe(opts, scram):
                                                            opts.oneEventMode,
                                                            opts.eventsPerLumi,
                                                            opts.maxRuntime)
-    ret = scram(command_, runtimeDir = os.getcwd(), logName='scramOutput.log')
```

Comment:
`scramOutput.log` is used [here](https://github.com/dmwm/CRABServer/blob/6321853aecb08ddf53890d23b8eff60a79d2cdb3/scripts/CMSRunAnalysis.sh#L188) but the current code is not creating it AFAICT.
Do we need something like [this](https://github.com/dmwm/CRABServer/pull/5887/files#diff-b4df3d7055437f8d2eeafa0b449218d5R653) here as well?

Reply:
yeah.. I did not test with scriptExe and clearly missed this. I new that testing was needed though. Let's fix this before testing though.

Reply:
I can't find that liner with `scramOutput.log` maybe you were looking at some previsous version.,

Reply:
Done

Reply:
where?

Reply:
should be al done now